src/App.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div id="app">
+    <img alt="Vue logo" src="./assets/logo.png">
+   <Button/>
+  </div>
+</template>
+
+<script>
+import Button from './components/Parent.vue'
```

Comment:
user proper names shouldn't be /Parent.vue
also don't put component name same as native HTML elements such as <Button>, rename it same as in the assignment